third_party/toolchains/backwards_compatibility.bzl

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 The gRPC authors.
```

Comment:
since you're adding this file, I think we need to cherrypick this PR into an import and see if this works internally as well.